syntax = "proto3";

message InferenceResult {
    uint64 frame_number = 1;       // 数据帧的编号
    int32 frame_width = 2;         // 数据帧的宽度
    int32 frame_height = 3;        // 数据帧的高度
    int32 frame_channels = 4;      // 数据帧的通道数
    bytes frame_raw_data = 5;      // 数据帧数据
    string publish_by = 6;         // 数据发布方
    bytes inference_results = 7;   // 检测结果
}
